velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -431,7 +431,7 @@ VectorPtr VectorFuzzer::fuzzConstant(const TypePtr& type, vector_size_t size) {
   // generated inner vector.
 
   // Inner vector size can't be zero.
-  auto innerVectorSize = rand<uint32_t>(rng_) % opts_.vectorSize + 1;
+  auto innerVectorSize = rand<uint32_t>(rng_) % (opts_.vectorSize + 1) + 1;
```

Comment:
Nit : Add a comment here saying that vectorSize could be 0 so adding 1 to avoid mod 0.